# Note that this project takes place in R

The Nobel Prize has been among the most prestigious international awards since 1901. Each year, awards are bestowed in chemistry, literature, physics, physiology or medicine, economics, and peace. In addition to the honor, prestige, and substantial prize money, the recipient also gets a gold medal with an image of Alfred Nobel (1833 - 1896), who established the prize.

![](Nobel_Prize.png)

The Nobel Foundation has made a dataset available of all prize winners from the outset of the awards from 1901 to 2023. The dataset used in this project is from the Nobel Prize API and is available in the `nobel.csv` file in the `data` folder.

In this project, you'll get a chance to explore and answer several questions related to this prize-winning data. And we encourage you then to explore further questions that you're interested in!

In [75]:
# Loading in required libraries
library(tidyverse)
library(readr)
library(ggplot2)


In [76]:
#read in the data
nobel = read_csv("data/nobel.csv")
glimpse(nobel)
nobel

Warning message:
“One or more parsing issues, call `problems()` on your data frame for details,
e.g.:
  dat <- vroom(...)
  problems(dat)”
Rows: 1000 Columns: 18
── Column specification ────────────────────────────────────────────────────────
Delimiter: ","
chr  (14): category, prize, motivation, prize_share, laureate_type, full_nam...
dbl   (2): year, laureate_id
date  (2): birth_date, death_date

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


Rows: 1,000
Columns: 18
$ year                 <dbl> 1901, 1901, 1901, 1901, 1901, 1901, 1902, 1902, 1…
$ category             <chr> "Chemistry", "Literature", "Medicine", "Peace", "…
$ prize                <chr> "The Nobel Prize in Chemistry 1901", "The Nobel P…
$ motivation           <chr> "\"in recognition of the extraordinary services h…
$ prize_share          <chr> "1/1", "1/1", "1/1", "1/2", "1/2", "1/1", "1/1", …
$ laureate_id          <dbl> 160, 569, 293, 462, 463, 1, 161, 571, 294, 464, 4…
$ laureate_type        <chr> "Individual", "Individual", "Individual", "Indivi…
$ full_name            <chr> "Jacobus Henricus van 't Hoff", "Sully Prudhomme"…
$ birth_date           <date> 1852-08-30, 1839-03-16, 1854-03-15, 1828-05-08, …
$ birth_city           <chr> "Rotterdam", "Paris", "Hansdorf (Lawice)", "Genev…
$ birth_country        <chr> "Netherlands", "France", "Prussia (Poland)", "Swi…
$ sex                  <chr> "Male", "Male", "Male", "Male", "Male", "Male", "…
$ organization_n

year,category,prize,motivation,prize_share,laureate_id,laureate_type,full_name,birth_date,birth_city,birth_country,sex,organization_name,organization_city,organization_country,death_date,death_city,death_country
<dbl>,<chr>,<chr>,<chr>,<chr>,<dbl>,<chr>,<chr>,<date>,<chr>,<chr>,<chr>,<chr>,<chr>,<chr>,<date>,<chr>,<chr>
1901,Chemistry,The Nobel Prize in Chemistry 1901,"""in recognition of the extraordinary services he has rendered by the discovery of the laws of chemical dynamics and osmotic pressure in solutions""",1/1,160,Individual,Jacobus Henricus van 't Hoff,1852-08-30,Rotterdam,Netherlands,Male,Berlin University,Berlin,Germany,1911-03-01,Berlin,Germany
1901,Literature,The Nobel Prize in Literature 1901,"""in special recognition of his poetic composition, which gives evidence of lofty idealism, artistic perfection and a rare combination of the qualities of both heart and intellect""",1/1,569,Individual,Sully Prudhomme,1839-03-16,Paris,France,Male,NA,NA,NA,1907-09-07,Châtenay,France
1901,Medicine,The Nobel Prize in Physiology or Medicine 1901,"""for his work on serum therapy, especially its application against diphtheria, by which he has opened a new road in the domain of medical science and thereby placed in the hands of the physician a victorious weapon against illness and deaths""",1/1,293,Individual,Emil Adolf von Behring,1854-03-15,Hansdorf (Lawice),Prussia (Poland),Male,Marburg University,Marburg,Germany,1917-03-31,Marburg,Germany
1901,Peace,The Nobel Peace Prize 1901,NA,1/2,462,Individual,Jean Henry Dunant,1828-05-08,Geneva,Switzerland,Male,NA,NA,NA,1910-10-30,Heiden,Switzerland
1901,Peace,The Nobel Peace Prize 1901,NA,1/2,463,Individual,Frédéric Passy,1822-05-20,Paris,France,Male,NA,NA,NA,1912-06-12,Paris,France
1901,Physics,The Nobel Prize in Physics 1901,"""in recognition of the extraordinary services he has rendered by the discovery of the remarkable rays subsequently named after him""",1/1,1,Individual,Wilhelm Conrad Röntgen,1845-03-27,Lennep (Remscheid),Prussia (Germany),Male,Munich University,Munich,Germany,1923-02-10,Munich,Germany
1902,Chemistry,The Nobel Prize in Chemistry 1902,"""in recognition of the extraordinary services he has rendered by his work on sugar and purine syntheses""",1/1,161,Individual,Hermann Emil Fischer,1852-10-09,Euskirchen,Prussia (Germany),Male,Berlin University,Berlin,Germany,1919-07-15,Berlin,Germany
1902,Literature,The Nobel Prize in Literature 1902,"""the greatest living master of the art of historical writing, with special reference to his monumental work, <I>A history of Rome</I>""",1/1,571,Individual,Christian Matthias Theodor Mommsen,1817-11-30,Garding,Schleswig (Germany),Male,NA,NA,NA,1903-11-01,Charlottenburg,Germany
1902,Medicine,The Nobel Prize in Physiology or Medicine 1902,"""for his work on malaria, by which he has shown how it enters the organism and thereby has laid the foundation for successful research on this disease and methods of combating it""",1/1,294,Individual,Ronald Ross,1857-05-13,Almora,India,Male,University College,Liverpool,United Kingdom,1932-09-16,Putney Heath,United Kingdom


There are 1,000 rows in this dataset.

# Q1
What is the most commonly awarded gender and birth country? Storing the string answers as `top_gender` and `top_country`.

In [77]:
#Q1

#count # sex-birth_country pairs
    #there are 154 such pairs when accounting for missing values
q1_data <- nobel %>%
    group_by(sex, birth_country) %>%
    summarize(num = n()) %>%
    ungroup() %>%
    arrange(desc(num))
q1_data

#assign specified variables
top_gender <- q1_data[1,]$sex
top_country <- q1_data[1,]$birth_country

`summarise()` has grouped output by 'sex'. You can override using the `.groups`
argument.


sex,birth_country,num
<chr>,<chr>,<int>
Male,United States of America,274
Male,United Kingdom,91
Male,Germany,65
Male,France,52
NA,NA,30
Male,Japan,28
Male,Sweden,28
Male,Canada,19
Male,Netherlands,19


Males in the USA have won more Nobel prizes than any other 'sex'-'birth_country' pair; 274. They account for 27.4% of all Nobel prizes in this dataset.

# Q2
What decade had the highest proportion of US-born winners? Store this as an integer called `max_decade_usa`.

In [78]:
#Q2

#need to make a new column denoting the decade of the year, which spans from 1901-2023
data_mod_q2 <- nobel %>%
    mutate(decade = case_when(year<1910 ~ "1900-1909", year>1909 & year<1920 ~ "1910-1919", 
                             year>1919 & year<1930 ~ "1920-1929", year>1929 & year<1940 ~ "1930-1939", 
                             year>1939 & year<1950 ~ "1940-1949", year>1949 & year<1960 ~ "1950-1959",
                             year>1959 & year<1970 ~ "1960-1969", year>1969 & year<1980 ~ "1970-1979", 
                             year>1979 & year<1990 ~ "1980-1989", year>1989 & year<2000 ~ "1990-1999",
                             year>1999 & year<2010 ~ "2000-2009", year>2009 & year<2020 ~ "2010-2019", year>2019 ~ "2020-2029"))
#use this modified data for the purposes of Q2
q2_data <- data_mod_q2 %>%
    filter(birth_country == "United States of America") %>%
    #are 291 observations of laureates from the US
    group_by(decade) %>%
    #obtain # & proportion of such laureates per decade
    summarize(num = n(), prop = n()/291) %>%
    ungroup() %>%
    #sort in descending order
    arrange(desc(num)) %>%
    #check to ensure that the new 'prop' column sums to 100% -- GOOD
    mutate(sum_prop=sum(prop))
q2_data

#assign the specified variable
max_decade_usa <- as.integer(str_replace(q2_data[1,]$decade, "-[0-9]{4}", ""))
max_decade_usa

decade,num,prop,sum_prop
<chr>,<int>,<dbl>,<dbl>
2000-2009,52,0.178694158,1
1990-1999,42,0.144329897,1
2010-2019,38,0.130584192,1
1970-1979,33,0.113402062,1
1980-1989,31,0.106529210,1
1950-1959,21,0.072164948,1
1960-1969,21,0.072164948,1
2020-2029,18,0.061855670,1
1930-1939,14,0.048109966,1


[1] 2000

The 2000-2009 decade accounts for the most US-born Nobel winners; 52. They account for about 18% of the 291 US-born observations.

# Q3
What decade and category pair had the highest proportion of female laureates? Store this as a list called `max_female_list` where the decade is the names of the values are `decade` and `category`.

In [79]:
#Q3
    #this question feeds off the data from Q2, can use "data_mod_q2"

q3_data <- data_mod_q2 %>%
    filter(sex == "Female") %>%
    #there are only 65 Female-laureate observations in the dataset
    group_by(decade, category) %>%
    #obtain # & proportion of such laureates per decade, category
    summarize(num = n(), prop = n()/65) %>%
    ungroup() %>%
    #sort in descending order
    arrange(desc(num)) %>%
    #check that all 'prop' values add to 100% -- GOOD
    mutate(sum_prop = sum(prop))
q3_data

#assign specified variable
max_female_list <- list(as.integer(str_replace(q3_data[1,]$decade, "-[0-9]{4}", "")), q3_data[1,]$category)
max_female_list

`summarise()` has grouped output by 'decade'. You can override using the
`.groups` argument.


decade,category,num,prop,sum_prop
<chr>,<chr>,<int>,<dbl>,<dbl>
2010-2019,Peace,5,0.07692308,1
2000-2009,Medicine,4,0.06153846,1
1970-1979,Peace,3,0.04615385,1
1980-1989,Medicine,3,0.04615385,1
1990-1999,Literature,3,0.04615385,1
1990-1999,Peace,3,0.04615385,1
2000-2009,Literature,3,0.04615385,1
2010-2019,Literature,3,0.04615385,1
2020-2029,Chemistry,3,0.04615385,1


[[1]]
[1] 2010

[[2]]
[1] "Peace"

The most popular 'decade'-'category' pair of female Nobel prize winners was 2010-2019 & Peace prizes. There were five such observations, which account for about 7.5% of the Nobel prizes won by females.

# Q4
Who was the first woman to receive a Nobel Prize, and in what category? Save your string answers as `first_woman_name` and `first_woman_category`.

In [80]:
#Q4

q4_data <- nobel %>%
    filter(sex == "Female") %>%
    #sort chronologically over time
    arrange(year) %>%
    #select variables of relevance
    select(year:prize, prize_share, full_name:sex)
q4_data

#assigned specified variables
first_woman_name <- q4_data[1,]$full_name
first_woman_name
first_woman_category <- q4_data[1,]$category
first_woman_category

year,category,prize,prize_share,full_name,birth_date,birth_city,birth_country,sex
<dbl>,<chr>,<chr>,<chr>,<chr>,<date>,<chr>,<chr>,<chr>
1903,Physics,The Nobel Prize in Physics 1903,1/4,"Marie Curie, née Sklodowska",1867-11-07,Warsaw,Russian Empire (Poland),Female
1905,Peace,The Nobel Peace Prize 1905,1/1,"Baroness Bertha Sophie Felicita von Suttner, née Countess Kinsky von Chinic und Tettau",1843-06-09,Prague,Austrian Empire (Czech Republic),Female
1909,Literature,The Nobel Prize in Literature 1909,1/1,Selma Ottilia Lovisa Lagerlöf,1858-11-20,Mårbacka,Sweden,Female
1911,Chemistry,The Nobel Prize in Chemistry 1911,1/1,"Marie Curie, née Sklodowska",1867-11-07,Warsaw,Russian Empire (Poland),Female
1926,Literature,The Nobel Prize in Literature 1926,1/1,Grazia Deledda,1871-09-27,"Nuoro, Sardinia",Italy,Female
1928,Literature,The Nobel Prize in Literature 1928,1/1,Sigrid Undset,1882-05-20,Kalundborg,Denmark,Female
1931,Peace,The Nobel Peace Prize 1931,1/2,Jane Addams,1860-09-06,"Cedarville, IL",United States of America,Female
1935,Chemistry,The Nobel Prize in Chemistry 1935,1/2,Irène Joliot-Curie,1897-09-12,Paris,France,Female
1938,Literature,The Nobel Prize in Literature 1938,1/1,Pearl Buck,1892-06-26,"Hillsboro, WV",United States of America,Female


[1] "Marie Curie, née Sklodowska"

[1] "Physics"

Marie Curie was the first woman to win a Nobel prize. She won her first, & first for all women, in 1903 in Physics. She won a 2nd Nobel prize, the fourth for ll women, in 1911 in Chemistry.

# Q5
Which individuals or organizations have won multiple Nobel Prizes throughout the years? Store the full names in a list named `repeats`.

In [81]:
#Q5

q5_data <- nobel %>%
    group_by(laureate_id) %>%
    #count # observations--# Nobel prize--per 'laureate_id'
    summarize(num_nobels = n()) %>%
    ungroup() %>%
    #filter for 'laureate_id's with multiple observations -- are 7 such id's
    filter(num_nobels>1) %>%
    #sort in descending order
    arrange(desc(num_nobels)) %>%
    #use a join to get other information (i.e. name, organization, ...)
    inner_join(orig_data, by=c("laureate_id"))
q5_data

#assign specified variable
repeats <- as.list(unique(q5_data$full_name))
repeats

laureate_id,num_nobels,year,category,prize,motivation,prize_share,laureate_type,full_name,birth_date,birth_city,birth_country,sex,organization_name,organization_city,organization_country,death_date,death_city,death_country
<dbl>,<int>,<dbl>,<chr>,<chr>,<chr>,<chr>,<chr>,<chr>,<date>,<chr>,<chr>,<chr>,<chr>,<chr>,<chr>,<date>,<chr>,<chr>
482,3,1917,Peace,The Nobel Peace Prize 1917,NA,1/1,Organization,Comité international de la Croix Rouge (International Committee of the Red Cross),NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
482,3,1944,Peace,The Nobel Peace Prize 1944,NA,1/1,Organization,Comité international de la Croix Rouge (International Committee of the Red Cross),NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
482,3,1963,Peace,The Nobel Peace Prize 1963,NA,1/2,Organization,Comité international de la Croix Rouge (International Committee of the Red Cross),NA,NA,NA,NA,NA,NA,NA,NA,NA,NA
6,2,1903,Physics,The Nobel Prize in Physics 1903,"""in recognition of the extraordinary services they have rendered by their joint researches on the radiation phenomena discovered by Professor Henri Becquerel""",1/4,Individual,"Marie Curie, née Sklodowska",1867-11-07,Warsaw,Russian Empire (Poland),Female,NA,NA,NA,1934-07-04,Sallanches,France
6,2,1911,Chemistry,The Nobel Prize in Chemistry 1911,"""in recognition of her services to the advancement of chemistry by the discovery of the elements radium and polonium, by the isolation of radium and the study of the nature and compounds of this remarkable element""",1/1,Individual,"Marie Curie, née Sklodowska",1867-11-07,Warsaw,Russian Empire (Poland),Female,Sorbonne University,Paris,France,1934-07-04,Sallanches,France
66,2,1956,Physics,The Nobel Prize in Physics 1956,"""for their researches on semiconductors and their discovery of the transistor effect""",1/3,Individual,John Bardeen,1908-05-23,"Madison, WI",United States of America,Male,University of Illinois,"Urbana, IL",United States of America,1991-01-30,"Boston, MA",United States of America
66,2,1972,Physics,The Nobel Prize in Physics 1972,"""for their jointly developed theory of superconductivity, usually called the BCS-theory""",1/3,Individual,John Bardeen,1908-05-23,"Madison, WI",United States of America,Male,University of Illinois,"Urbana, IL",United States of America,1991-01-30,"Boston, MA",United States of America
217,2,1954,Chemistry,The Nobel Prize in Chemistry 1954,"""for his research into the nature of the chemical bond and its application to the elucidation of the structure of complex substances""",1/1,Individual,Linus Carl Pauling,1901-02-28,"Portland, OR",United States of America,Male,California Institute of Technology (Caltech),"Pasadena, CA",United States of America,1994-08-19,"Big Sur, CA",United States of America
217,2,1962,Peace,The Nobel Peace Prize 1962,NA,1/1,Individual,Linus Carl Pauling,1901-02-28,"Portland, OR",United States of America,Male,California Institute of Technology (Caltech),"Pasadena, CA",United States of America,1994-08-19,"Big Sur, CA",United States of America


[[1]]
[1] "Comité international de la Croix Rouge (International Committee of the Red Cross)"

[[2]]
[1] "Marie Curie, née Sklodowska"

[[3]]
[1] "John Bardeen"

[[4]]
[1] "Linus Carl Pauling"

[[5]]
[1] "Frederick Sanger"

[[6]]
[1] "Office of the United Nations High Commissioner for Refugees (UNHCR)"

[[7]]
[1] "K. Barry Sharpless"

[[8]]
[1] "Barry Sharpless"

In this dataset, there are seven unique individuals/organizations that won multiple Nobel prizes. Six of them won two, & the Comité international de la Croix Rouge (International Committee of the Red Cross) won three. Note that the `repeats` list contains eight unique elements, which is because the dataset has an inconsistency in the 'full_name' column regarding Barry Sharpless. This individual's 'laureate_id' corresponds to two different 'full_name's.